datamap/mv/plan/src/main/scala/org/apache/carbondata/mv/plans/modular/basicOperators.scala

```diff
@@ -39,6 +39,21 @@ case class GroupBy(
     flagSpec: Seq[Seq[Any]],
     dataMapTableRelation: Option[ModularPlan] = None) extends UnaryNode with Matchable {
   override def output: Seq[Attribute] = outputList.map(_.toAttribute)
+
```

Comment:
please add comments, same applicable for next one more method also

Reply:
done